Tests/xDnsRecord.Tests.ps1

```diff
@@ -11,22 +11,24 @@ Set-StrictMode -Version latest
 
 $RepoRoot = (Resolve-Path $PSScriptRoot\..).Path
 
-$ModuleName = "MSFT_xDnsARecord"
+$ModuleName = "MSFT_xDnsRecord"
 Import-Module (Join-Path $RepoRoot "DSCResources\$ModuleName\$ModuleName.psm1")
 Import-Module DnsServer
 
-Describe "xDnsARecord" {
+Describe "xDnsRecord" {
     InModuleScope $ModuleName {
         $testPresentParams = @{
             Name = "test"
             Zone = "contoso.com"
             Target = "192.168.0.1"
+            Type = "ARecord"
             Ensure = "Present"
         }
         $testAbsentParams = @{
             Name = $testPresentParams.Name
             Zone = $testPresentParams.Zone
             Target = $testPresentParams.Target
+            Type = $testPresentParams.Type
             Ensure = "Absent"
         }
```

Comment:
Should have a test that Get-targetresource correctly returns the current type of the record.